lib/utils/__tests__/getDimension.test.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const getDimension = require('../getDimension');
+const valueParser = require('postcss-value-parser');
+
+it('getDimension', () => {
+	expect(getDimension().number).toBeNull();
+	expect(getDimension().unit).toBeNull();
+	expect(getDimension({}).number).toBeNull();
+	expect(getDimension({}).unit).toBeNull();
+
+	// testing Dimension.unit
+	expect(getDimension(valueParser('1.1s').nodes[0]).unit).toBe('s');
```

Comment:
[trivial] [`.toEqual()`](https://jestjs.io/docs/expect#toequalvalue) can reduce repetations as below:

```suggestion
	expect(getDimension(valueParser('1.1s').nodes[0])).toEqual({ unit: 's', number: '1.1' });
```

But keeping the code as-is has no problems. 👍🏼